packages/glsp-playwright-core/src/debugger/extractor.ts

```diff
@@ -0,0 +1,35 @@
+import { Locator } from '@playwright/test';
+import { SVGMetadata } from '~/glsp/graph/svg-metadata-api';
+import { collectLocators } from '../utils/playwright.utils';
+
```

Comment:
We definitely need more documentation on helper functions and interfaces like these.